tests/test-core/jqUnit/js/jqUnit-browser.js

```diff
@@ -132,7 +132,7 @@ jqUnit.assertNode = function (message, expected, node) {
             messageExt = " - node name";
         }
         else if (key === "nodeText") {
-            attr = fluid.dom.getElementText(node).trim();
+            attr = node.innerText;
```

Comment:
Looks like this is causing the issues with Paged Table tests in Safari. for example it's returning `"dog "`. Probably just need to add `trim` back.